Source/missiles.cpp

```diff
@@ -198,7 +198,7 @@ static bool CheckBlock(Point from, Point to)
 	return false;
 }
 
-static inline bool InDungeonBounds(Point position)
+static inline bool InDungeonBounds(const Point position)
 {
 	return position.x > 0 && position.x < MAXDUNX && position.y > 0 && position.y < MAXDUNY;

```

Comment:
While this isn't an issue with the changes made in this PR this check doesn't quite match the function name. The last row/col is considered in bounds but the first row/col isn't. I would like to fix this eventually even if in practice it doesn't change anything 😅 . Bringing this up because ProcessMissiles() relies on the current behaviour to compensate for missing checks in the missile logic functions.